scss/modules/content-types.scss

```diff
@@ -354,28 +358,48 @@
                 .num {
                     color: $grayLight;
                     padding-right: 15px;
+                    display: table-cell;
+                    width: 1%; // make cell fit content
```

Comment:
What do you want me to describe in more detail? The only reason for the `1%`, is that the table-cell should fit its content. I don't really know what to add to this. This a common pattern and has to do with the table-layout and how the browser renders it.